os/arch/xtensa/src/esp32/Kconfig

```diff
@@ -118,8 +122,16 @@ config ESP32_SPI1
 config ESP32_SPI2
 	bool "SPI 2"
 	default n
+	select ESP32_SPI
```

Comment:
Please add this in SPI0 and SPI1 as well.

Reply:
Fixed!